mono/eglib/glib.h

```diff
@@ -381,6 +382,8 @@ gint         g_printf          (gchar const *format, ...) G_ATTR_FORMAT_PRINTF(1
 gint         g_fprintf         (FILE *file, gchar const *format, ...) G_ATTR_FORMAT_PRINTF(2, 3);
 gint         g_sprintf         (gchar *string, gchar const *format, ...) G_ATTR_FORMAT_PRINTF(2, 3);
 gint         g_snprintf        (gchar *string, gulong n, gchar const *format, ...) G_ATTR_FORMAT_PRINTF(3, 4);
+gint         g_async_safe_printf (gchar const *format, ...) G_ATTR_FORMAT_PRINTF(1, 2);
```

Comment:
Make this functions `static inline` declared in this header to reduce chances of modifying the performance footprint.